t/mifir.t

```diff
@@ -32,11 +32,38 @@ is mifir_concat({
 
-is Finance::MIFIR::CONCAT::_process_name('Аркадий'),       'arkad', 'russian check';
-is Finance::MIFIR::CONCAT::_process_name('Стругацкий'), 'strug', 'russian check';
-is Finance::MIFIR::CONCAT::_process_name('АЙЗЕК'),           'aizek', 'russian check';
-is Finance::MIFIR::CONCAT::_process_name('Азимов'),         'azimo', 'russian check';
-is Finance::MIFIR::CONCAT::_process_name('Бьёрн'),           'bern#', 'russian check';
-is Finance::MIFIR::CONCAT::_process_name('Страуструп'), 'strau', 'russian check';
-
+is Finance::MIFIR::CONCAT::_process_name('ßŚáŹł'), 'ssazl', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ĄŴÇĎŇ'), 'awcdn', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('êŽǍţš'), 'ezats', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ěęŐÒñ'), 'eeoon', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ĝœëşĚ'), 'goese', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('åÝćųĺ'), 'aycul', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ĵăżÔť'), 'jazot', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ÄĥÆÜĘ'), 'ahaue', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ÁďľĐą'), 'adlda', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('àÕìÍÂ'), 'aoiia', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('řïůŰÖ'), 'riuuo', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('õŒØŁȚ'), 'ooolt', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ĽÃĤæè'), 'lahae', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('éĞğçț'), 'eggct', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('űŃģÿó'), 'ungyo', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ÊúĻčÅ'), 'eulca', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ĉâÀŮő'), 'caauo', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('îŨĹĢÙ'), 'iulgu', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ıŢÎýŝ'), 'itiys', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ŤŕòŲČ'), 'trouc', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('íśÏșÛ'), 'isisu', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('đûËŵĈ'), 'duewc', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ùüžĴð'), 'uuzjd', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ŜŠÈŔã'), 'ssera', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ĆŶäȘũ'), 'cyasu', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ÚŞǎöĂ'), 'usaoa', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('þŻÓøẞ'), 'tzoos', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ôļŘķň'), 'olrkn', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('źĶŷÉń'), 'zkyen', 'character check';
+is Finance::MIFIR::CONCAT::_process_name('ÌÞŸĜÑ'), 'itygn', 'character check';
```

Comment:
Hi, just commenting on Perl style here (as well as getting used to the codebase in general.)

Are the character check tests exhaustive and call all combinations of these characters?  If not, would it make sense to generate all these combinations and put into another file that this test script can loop over?

Reply:
Hi @zakame , I don't think it is necessary to test all combinations here. It is enough to make sure all characters in transliteration tables are replaced with the right character. We don't care about combinations. But I extract all strings into a variable now.

Reply:
Cool, no worries :+1: